modules/ghservice/templates/apache/app_admin.conf.erb

```diff
@@ -0,0 +1,95 @@
+    ##
```

Comment:
All extras or changes in these configuration files should be put in a PR to grasshopper-ui as well so we don't lose track of them.